client/post-editor/editor-term-selector/index.jsx

```diff
@@ -77,7 +80,7 @@ class EditorTermSelector extends Component {
 export default connect(
 	( state ) => {
 		const siteId = getSelectedSiteId( state );
-		const postId = getEditorPostId( state );
+		const postId = getEditorPostId( state ) || '';
```

Comment:
Out of curiosity, does this branch not work if you don't provide this fallback string? Since the middleware is simply checking for the presence of a key, even if `null`, and `editPost` gracefully handles the `null` `postId` case, could we remove this? Might simplify things a bit.